help/marking-messages-as-read.md

```diff
@@ -22,8 +22,8 @@ web/desktop app.
 - **Only in conversation views**: In Zulip, a **conversation** is a [direct
   message](/help/direct-messages) thread (one-on-one or with a group), or a
   [topic in a stream](/help/streams-and-topics). This option makes it convenient
-  to preview new messages in a stream, or skim [All
-  messages](/help/all-messages), and later read each topic in detail.
+  to preview new messages in a stream, or skim [Combined
+  feed](/help/combined-feed), and later read each topic in detail.
```

Comment:
I think "skim your [combined feed]" would be a better way to write this sentence given the new language -- the "your" feels more natural here than with the previous name.